examples/kube-stack-private/README.md

```diff
@@ -0,0 +1,103 @@
+```bash
+cp contrib/aws_inventory/kubespray-aws-inventory.py inventory/mycluster/
+```
+6. You need to define the following three variables for this script to work properly:
+```bash
+export AWS_ACCESS_KEY_ID="xxxxx"
+export AWS_SECRET_ACCESS_KEY="yyyyy"
+export REGION="<region where you put the cluster>"
+```
+7. To test that you properly set things up in the previous step run the following command. It should print out your inventory in the terminal, listing all the nodes along with what role(s) they have in the cluster: `etcd`,`master` and/or `node`
+```bash
+python inventory/mycluster/kubespray-aws-inventory.py --list
+```
+8. Next you need to set some parameters in the file `inventory/mycluster/group.vars/all.yml`. Uncomment these variables and/or set the values as directed below. Note that these variables are scattered throughout the file so you have to look for them. Note as well you should review the entire file since there may be other variables you might like to change:
+```
+bootstrap_os: coreos
+apiserver_loadbalancer_domain_name: "<see note below on how to find elb name which should put here between quotes>"
+# since we use an external load balancer we need to set local lb to false
+loadbalancer_apiserver_localhost: false
+cloud_provider: aws
+etcd_memory_limit: 0
+```
+9. You can find the DNS name for the API server load balancer by running `terraform show` and searching for `module.kube-cluster.aws_elb.kube-controllers`. One of the parameters listed under this is `dns_name`. Copy the value into the appropriate place above.
+10. Delete the file `inventory/mycluster/hosts.ini` since we are using dynamic inventory.
+11. Next you need to set some parameters in the file `inventory/mycluster/group.vars/k8s-cluster.yml`. Uncomment these variables and/or set the values as directed below. Note that these variables are scattered throughout the file so you have to look for them. Note as well you should review the entire file since there may be other variables you might like to change:
+```
+kube_api_anonymous_auth: false
+enable_network_policy: true
+cluster_name: mycluster.local
+resolvconf_mode: host_resolvconf
+helm_enabled: true
+ingress_nginx_enabled: true
+kubeconfig_localhost: true
+kubectl_localhost: true
+```
+12. We are now ready to run the ansible playbooks. You need to add the `-b` flag so scripts can take on `sudo` privileges:
+```bash
+ansible-playbook -b -i inventory/atcluster/kubespray-aws-inventory.py  cluster.yml
+```
+13. Once Lubespray is finished you need to copy the generated `kubectl` cofiguration to a convenient location:
+```
+cp inventory/atcluster/artifacts/admin.conf ~/.kube
```

Comment:
in previous code examples you used the shell prompt prefix, but not down here. This should be consistent, either one way or the other.

Reply:
fixed